README.md

```diff
@@ -89,7 +89,7 @@ For installation and usage instructions, see: <https://fdc3.finos.org/docs/suppo
 If you are an existing individual or corporate user of the FDC3 standard, we would love to hear from you: just email [fdc3@finos.org](mailto:fdc3@finos.org) with details about how you are using the standard.
 
 - If you'd like to be listed as on the community page, please fill out the [Usage Form](https://share.hsforms.com/1gOo0_A70QqyahOygzryVQg1fux8).
-- If listing your logo publicly requires legal evaluation, you can reach out **privately** to [FDC3 Maintainers](mailto:fdc3-maintainers-private@finos.org).
+- If listing your logo publicly requires legal evaluation, you can reach out **privately** to [FDC3 x](mailto:fdc3-maintainers-private@finos.org).
```

Comment:
@robmoffat is that `FDC3 x` a typo? Shouldn't it be `FDC3 Maintainers` ?

Reply:
yes, probably me trying to cut something -good catch